dashboard/client/src/pages/job/index.tsx

```diff
@@ -100,29 +96,55 @@ const JobList = () => {
                   page.pageNo * page.pageSize,
                 )
                 .map(
-                  ({
-                    jobId = "",
-                    driverIpAddress,
-                    isDead,
-                    driverPid,
-                    startTime,
-                    endTime,
-                  }) => (
-                    <TableRow key={jobId}>
+                  (
+                    {
+                      job_id,
+                      submission_id,
+                      driver_info,
+                      type,
+                      status,
+                      start_time,
+                      end_time,
+                    },
+                    index,
+                  ) => (
+                    <TableRow key={job_id ?? submission_id ?? index}>
+                      <TableCell align="center">{job_id ?? "-"}</TableCell>
                       <TableCell align="center">
-                        <Link to={`/job/${jobId}`}>{jobId}</Link>
+                        {submission_id ?? "-"}
                       </TableCell>
-                      <TableCell align="center">{driverIpAddress}</TableCell>
-                      <TableCell align="center">{driverPid}</TableCell>
+                      <TableCell align="center">{status}</TableCell>
                       <TableCell align="center">
-                        {isDead ? "true" : "false"}
+                        {/* TODO(aguo): Also show logs for the job id instead
+                        of just the submission's logs */}
+                        {driver_info &&
+                        ipLogMap[driver_info.node_ip_address] ? (
+                          <Link
+                            to={`/log/${encodeURIComponent(
+                              ipLogMap[driver_info.node_ip_address],
+                            )}?fileName=${
+                              type === "DRIVER"
+                                ? job_id
+                                : `driver-${submission_id}`
+                            }`}
+                            target="_blank"
+                          >
+                            Log
+                          </Link>
+                        ) : (
+                          "-"
+                        )}
                       </TableCell>
                       <TableCell align="center">
-                        {dayjs(Number(startTime)).format("YYYY/MM/DD HH:mm:ss")}
+                        {dayjs(Number(start_time)).format(
```

Comment:
Can we also add a running_time column?  (could be done in the future PRs though). But I think that'd be pretty useful

Reply:
I'll do this in a follow-up..

Reply:
btw ping on this as a follow up

Reply:
Btw, I think we can do this after 2.0.